CONTRIBUTING.md

```diff
@@ -46,62 +47,15 @@ If you can find an issue that's the same as yours:
   and resolve the issue by adding a new comment with your details.
 
 ### Create a New Issue
+
 You can
 [create a new issue](https://github.com/openstenoproject/plover/issues/new).
-You can help us by copying this template:
 
 #### New Issue Template
-```markdown
-Classification: (Pick one of: Bug, Crash/Hang/Data Loss, Performance, UI/Usability, Feature (New), Enhancement)
-
-Reproducibility: (Pick one of: Always, Sometimes, Rarely, Unable, I Didn't Try, Not Applicable - N/A applies for Feature and Enhancement requests)
-
-## Summary
-Summary of the problem.
-
-## Steps to Reproduce
-1. I do this
-2. I do that
-
-## Expected Results
-3. This should happen.
-
-## Actual Results
-3. Instead, this happens. :(
-
-## Version
-Plover version ???.
-(Click the "About…" button in the main Plover window to see the version
-number.)
-
-Installed via: (download from openstenoproject.org, Homebrew, apt-get, …).
 
-## Notes
-Describe any debugging steps you've taken yourself.
-
-If you've found a workaround, provide it here.
-
-## Configuration
-OS:
-
-- If it's OS X,
-  go to Finder, Apple menu, "About This Mac".
-  - Alternatively, if you're comfortable with the Terminal, run
-    `sw_vers` and paste the output.
-
-- If it's Linux,
-  provide information on your distro and any customization
-  relevant to Plover's operation (desktop environment, window manager, ???).
-  - uname -a output might also be useful.
-
-- If it's Windows,
-  press the Windows key + PauseBreak key together in order to bring up
-  the Windows version information.
-```
-
-If helpful, submit screenshots of the issue to help debug.
-The Plover.log or strokes.log may also be useful.
+When creating an issue, you should be presented with the [issue template](.github/ISSUE_TEMPLATE.md).
```

Comment:
If think we can just remove this section, no need to mention the template, just the link for a new issue is enough, no?